src/sharedData/components/SharedDataEntryBase.js

```diff
@@ -169,16 +171,23 @@ const SharedDataEntryBase = props => {
         container
         spacing={1}
         alignItems="center"
-        sx={{ fontSize: 14, color: 'gray.dark1', p: 1 }}
+        sx={{
+          fontSize: 14,
+          color: 'gray.dark1',
+          p: 1,
+        }}
       >
         <Grid
           item
-          xs={isEditingName ? 12 : 8}
+          xs={isEditingName ? 12 : 7}
           md={5}
           order={{ xs: 2, md: 2 }}
           component={StackRow}
         >
-          <EntryTypeIcon resourceType={dataEntry.resourceType} />
+          <EntryTypeIcon
+            resourceType={dataEntry.resourceType}
+            style={isSmall ? { marginTop: '-1em', paddingRight: '5px' } : {}}
```

Comment:
I don't understand this approach -- I tried these changes and it doesn't seem to line up the top of the icon with the top of the title's first line of text. (The title can also be a variable number of lines long.)

Would we instead want `alignItems: 'flex-start'` somewhere, maybe with some padding?

Reply:
Good idea. I've used flex-start.